src/battle-scene.ts

```diff
@@ -1407,7 +1416,7 @@ export default class BattleScene extends SceneBase {
       }
       /**
        * Override battles into single only if not fighting with trainers.
-       * @see {@link https://github.com/pagefaultgames/pokerogue/issues/1948 | GitHub Issue #1948}
+       * @see {@link https://github.com/pagefaultgames/pokerogue/issues/1948 GitHub Issue #1948}
```

Comment:
```suggestion
       * @see {@link https://github.com/pagefaultgames/pokerogue/issues/1948 | GitHub Issue #1948}
```
You're supposed to have the `|` if you want to "rename" a `@link[code]` according to the spec.
![image](https://github.com/user-attachments/assets/7f1a9ba6-705b-49aa-95a4-1b48af9c177f)
https://tsdoc.org/play/